compiler/compiler/src/main/java/org/robovm/compiler/util/ToolchainUtil.java

```diff
@@ -253,6 +254,10 @@ public static void actool(Config config, File partialInfoPlist, File inDir, File
             }
         }
 
+        //Prepares for incremental build - not applied at this time.
```

Comment:
could you please add comment what it affect to and how it works

Reply:
Added a longer comment here.